pagopa/pn-frontend
# use single contact compoent into SpecialContactElem - pn-12069

## Short description
Use single contact compoent into SpecialContactElem

## List of changes proposed in this pull request
- Updated PecContactItem, EmailContactItem, SmsContactItem to manage special contact case
- Integrated into SpecialContactElem single contact components
- Updated tests

## How to test
Login with pf/pg -> go to Recapiti page -> add/edit/remove special contacts and check that everything is ok
